src/BenchmarkDotNet.Core/Diagnosers/CompositeDiagnoser.cs

```diff
@@ -19,13 +19,13 @@ public CompositeDiagnoser(params IDiagnoser[] diagnosers)
 
         public IColumnProvider GetColumnProvider() 
             => new CompositeColumnProvider(diagnosers.Select(d => d.GetColumnProvider()).ToArray());
-
+#if !UAP
```

Comment:
The Process class is available as ref nuget package. But there is no aop runtime implementation of Process and this causes app build failure with smth like "unresolved class" error.